modules/tile-converter/src/converter-cli.ts

```diff
@@ -331,6 +346,9 @@ function parseOptions(args: string[]): TileConversionOptions {
         case '--output':
           opts.output = getStringValue(index, args);
           break;
+        case '--tiles-version':
```

Comment:
Do we need this option? I suggest get rid of b3dm. It is a legacy format. People can use an older converter version if they want to have 1.0 as an output